themes/custom/lex/scss/regions/_footer.scss

```diff
@@ -66,6 +66,7 @@ body footer {
     &:hover {
       color: color(yellow);
     }
+    text-decoration: underline;
```

Comment:
Can we make it...
  
  a {
    color: color(white);
    text-decoration: underline;
    
    &:hover {
      color: color(yellow);
    }
   }

Reply:
@ChrispyAPAX